screens/auth/AuthScreen.js

```diff
@@ -24,14 +23,18 @@ export default class AuthScreen extends React.Component {
         this.setState({[field]: value});
     }
 
-    render() {
+    render(url) {
```

Comment:
It's better practice not having the `render` function take in an argument, you can instead use a state variable for this.